### PR TITLE
Feature/apd 2877 logout

### PR DESCRIPTION
**Pull request by mirceamarcu89**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,6 +32,8 @@
     "istanbul": "^0.4.5",
     "mocha": "^3.4.2",
     "prettier": "^1.5.3",
+    "sinon": "^3.3.0",
```
Comment (paul-pop) [inline package.json]: Any reason we are not using jest and jest mocks?

File: test/unit.js [hunk 2]
```diff
@@ -1,16 +1,99 @@
...
+  });
+});
+
 describe("refreshToken", () => {
   it("should be a function", () => {
     let client = new ANDGatewayClient();
     expect(typeof client.refreshToken).to.equal("function");
   });
 });
+
```
Comment (ellieCrome) [inline test/unit.js]: all of these tests below are technically part of ollies story

File: test/unit.js [hunk 3]
```diff
@@ -1,16 +1,99 @@
...
+const chai = require("chai");
+const sinon = require("sinon");
+const sinonChai = require("sinon-chai");
+const expect = chai.expect;
+chai.use(sinonChai);
+
 const ANDGatewayClient = require("../lib/and-sso-js.js");
 
+global.window = { location: {} };
+
```
Comment (ellieCrome) [inline test/unit.js]: whats the point of this?

File: package.json [hunk 4]
```diff
@@ -46,5 +47,22 @@
   "bugs": {
     "url": "https://github.com/ANDigital/and-sso-js/issues"
   },
-  "homepage": "https://github.com/ANDigital/and-sso-js#readme"
+  "homepage": "https://github.com/ANDigital/and-sso-js#readme",
+  "jest": {
+    "modulePaths": [
```
Comment (paul-pop) [inline package.json]: Some of these are defaults, look at the jest documentation

File: test/unit.js [hunk 5]
```diff
@@ -1,16 +1,99 @@
-const { expect } = require("chai");
-const ANDGatewayClient = require("../lib/and-sso-js.js");
+const ANDGatewayClient = require("../src/entry.js");
+
+describe("ANDGatewayClient", () => {
+  it("should be an instantiable class", () => {
+    expect(1).toBe(1);
```
Comment (paul-pop) [inline test/unit.js]: ?!?!?!?!?!?!?!?!?!?!?!?!

**Pull request merged**